sentry_dart_plugin/lib/src/utils/log.dart

```diff
@@ -0,0 +1,105 @@
+import 'dart:io';
+import 'package:ansicolor/ansicolor.dart';
+
+int numberOfAllTasks = 11;
+
+class Log {
+  static AnsiPen red = AnsiPen()..red(bold: true);
+  static AnsiPen yellow = AnsiPen()..yellow(bold: true);
+  static AnsiPen green = AnsiPen()..green(bold: true);
+  static AnsiPen blue = AnsiPen()..blue(bold: true);
+  static AnsiPen gray05 = AnsiPen()..gray(level: 0.5);
+  static AnsiPen gray09 = AnsiPen()..gray(level: 0.9);
+  static int _numberOfTasksCompleted = 0;
+  static int lastMessageLength = 0;
+
+  /// Log with colors.
+  Log();
+
+  /// Information log with `white` color
+  static void info(String message) => _write(message, withColor: gray09);
+
+  /// Error log with `red` color
+  static void error(String message) => _write(message, withColor: red);
+
+  /// Write `error` log and exit the program
+  static void errorAndExit(String message) {
+    error(message);
+    exit(0);
```

Comment:
This should be a non-zero exit code because of the error.
0 implies a successful exit which this isn't.